### PR TITLE
add iOS example module

### LINKED ISSUE #37: [Feature] | Add iOS example app.
**Which kind of problem this feature would address? Please describe.**
We don't have an iOS example app for Kotlin Multi platform, which is going to be a good addition for the library as we may need to make some adjustments for that platform.

**Describe the solution you'd like**
Create a new module in the project called iOS example app, and make its UI, creating a sample form using MVVM pattern. Do not implement FlowForms yet, that will be tackled in another Issue. 
The project should run correctly in an iOS phone or emulator.

**Additional context**
We can take the Android example app as an example.

### PR DESCRIPTION
**Pull request by tarruk**
#### Issue
*  closes #37

---

#### Description
* iOS example App module was added as a formView
* framework: SwiftUI

#### Notes
* Some UI odd behaviors were noticed when the app is built from androidStudio

#### Preview
- From Xcode: 

https://user-images.githubusercontent.com/36048605/198397609-4feb6438-2728-4fc3-bc49-dabaca3d185e.mov

- From Android Studio:

https://user-images.githubusercontent.com/36048605/198397644-c32fdef7-e39c-4559-ac31-c1fe35f859fa.mov

### CODE REVIEW

**Review by ximenaperez — APPROVED** [review]
great work! looks really good!

File: ExampleApp iOS/flowforms/FlowForms.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import SwiftUI
+import shared
+
+@main
+struct FlowForms: App {
```
Comment (hrodrick) [inline ExampleApp iOS/flowforms/FlowForms.swift:5]: What do you think about naming this struct and file `FlowFormsApp`

File: settings.gradle.kts [hunk 2]
```diff
@@ -1,2 +1,3 @@
 include(":FlowForms-Core", ":ExampleApp Android")
 rootProject.name = "FlowForms"
+include(":shared")
```
Comment (hrodrick) [inline settings.gradle.kts:3]: We can add `, :shared`  to the first include statement. What do you think?

File: shared/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+android {
+    namespace = "com.rootstrap.flowforms.shared"
+    compileSdk = 32
+    defaultConfig {
+        minSdk = 21
+        targetSdk = 32
+    }
+}
```
Comment (hrodrick) [inline shared/build.gradle.kts:56]: Please add an empty line at the end of this file

File: shared/src/androidMain/kotlin/com/rootstrap/flowforms/shared/Platform.kt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.rootstrap.flowforms.shared
+
+class AndroidPlatform : Platform {
+    override val name: String = "Android ${android.os.Build.VERSION.SDK_INT}"
+}
+
+actual fun getPlatform(): Platform = AndroidPlatform()
```
Comment (hrodrick) [inline shared/src/androidMain/kotlin/com/rootstrap/flowforms/shared/Platform.kt:7]: Please add an empty line at the end of this file

File: shared/src/commonMain/kotlin/com/rootstrap/flowforms/shared/Greeting.kt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package com.rootstrap.flowforms.shared
+
+class Greeting {
+    private val platform: Platform = getPlatform()
+
+    fun greeting(): String {
+        return "Hello, ${platform.name}!"
+    }
+}
```
Comment (hrodrick) [inline shared/src/commonMain/kotlin/com/rootstrap/flowforms/shared/Greeting.kt:9]: Please add an empty line at the end of this file

File: shared/src/commonMain/kotlin/com/rootstrap/flowforms/shared/Platform.kt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package com.rootstrap.flowforms.shared
+
+interface Platform {
+    val name: String
+}
+
+expect fun getPlatform(): Platform
```
Comment (hrodrick) [inline shared/src/commonMain/kotlin/com/rootstrap/flowforms/shared/Platform.kt:7]: Please add an empty line at the end of this file

File: shared/src/iosMain/kotlin/com/rootstrap/flowforms/shared/Platform.kt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+package com.rootstrap.flowforms.shared
+
+import platform.UIKit.UIDevice
+
+class IOSPlatform: Platform {
+    override val name: String = UIDevice.currentDevice.systemName() + " " + UIDevice.currentDevice.systemVersion
+}
+
+actual fun getPlatform(): Platform = IOSPlatform()
```
Comment (hrodrick) [inline shared/src/iosMain/kotlin/com/rootstrap/flowforms/shared/Platform.kt:9]: Please add an empty line at the end of this file

**Review by hrodrick — APPROVED** [review]
Looks good! I just left some comments for the Kotlin files but nothing else
Nice work!

**Review by hrodrick — APPROVED** [review]
Nice work!

**Pull request merged**